int8 mission_type